.gitignore

```diff
@@ -14,6 +14,8 @@ node_modules/
 .env.development.local
 .env.test.local
 .env.production.local
+/development
+
```

Comment:
What's this directory?

Reply:
This is just where I'm placing things like screenshots/screencaps to post here to git, I can make it public if you want!

Reply:
No need, just was curious what the folder was. Fine with me 👍🏻